packages/components/tree-v2/src/tree-node-content.vue

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
```

Comment:
use TypeScript file directly.
rename `tree-node-content.vue` to `tree-node-content.ts`